internal/database/mariadb/activity.go

```diff
@@ -208,9 +211,11 @@ func (s *SqlDatabase) CreateActivity(activity *entity.Activity) (*entity.Activit
 
 	query := `
 		INSERT INTO Activity (
```

Comment:
aren't we missing the updated_by field? This is also in the other db queries the case

Reply:
Do we need to insert it during creation of an object? is it possible that updatedBy will be given when the object is created in database?